CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+[4.0.0 - MM-DD-YY](https://github.com/cerner/cerner_tomcat/issues?milestone=TBD&state=closed)
```

Comment:
Not sure if this actually warrants a 4.0 bump... Technically by semver expectations it does, but it also seems a bit overkill considering it's such a tiny compatibility change in the big scheme of things. I also should have bumped metadata.rb if this should be bumped... Not sure what y'all think about it warranting a version bump or not though so going to leave it, and if y'all do want to bump it, I'll update metadata.rb too.

Reply:
If we remove it I'd say lets bump to 4.0, unless you'd want to keep java in and support both 1.x and 2.x